x-pack/platform/plugins/shared/onechat/public/application/components/tools/form/validation/index_search_tool_form_validation.ts

```diff
@@ -37,46 +31,7 @@ const indexSearchI18nMessages = {
   },
 };
 
-export const useIndexSearchToolFormValidationResolver = (): Resolver<IndexSearchToolFormData> => {
```

Comment:
suggestion from @dennis-tismenko from last week, we can use `zodResolver`, sp this is no longer needed